appengine/requests/README.md

```diff
@@ -0,0 +1,10 @@
+## App Engine Requests Docs Snippets
+
+These snippets demonstrate various aspects of App Engine Python request handling.
+
+<!-- auto-doc-link -->
```

Comment:
there's a script that can automatically populate these. I need to re-run it once we're done moving all samples.